catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -719,7 +719,15 @@ protected InputTransformer lookupTransformerReference(String namespaceUri)
       Collection<ServiceReference<InputTransformer>> transformerReference =
           bundleContext.getServiceReferences(
               InputTransformer.class, "(schema=" + namespaceUri + ")");
-      return bundleContext.getService(transformerReference.iterator().next());
+      if (transformerReference.isEmpty()) {
+        LOGGER.trace("Failed to find Input Transformer by schema : {}", namespaceUri);
+      } else {
+        LOGGER.trace(
+            "Found Input Transformer {} by schema {}",
+            transformerReference.iterator().next().getBundle().getSymbolicName(),
+            namespaceUri);
+        return bundleContext.getService(transformerReference.iterator().next());
```

Comment:
nit: could refactor `transformerReference.iterator().next()` into it's own variable

Reply:
Also could use `transformerReference.stream().findFirst().orElse(null)` if you don't want to do the iterator thing.